spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -113,15 +111,25 @@ def expect_retryable_error(error_class)
           end
 
           it 'already includes classification code and does not modify' do
-            VCR.use_cassette('evss/disability_compensation_form/submit_form_v2') do
-              VCR.use_cassette('mail_automation/mas_initiate_apcas_request') do
-                subject.perform_async(submission.id)
-                described_class.drain
-                mas_submission = Form526Submission.find(Form526JobStatus.last.form526_submission_id)
-                expect(mas_submission.form.dig('form526', 'form526',
-                                               'disabilities').first['classificationCode']).to eq '8935'
-              end
-            end
+            subject.perform_async(submission.id)
+            described_class.drain
+            mas_submission = Form526Submission.find(Form526JobStatus.last.form526_submission_id)
+            expect(mas_submission.form.dig('form526', 'form526',
+                                           'disabilities').first['classificationCode']).to eq '8935'
+          end
+        end
+
+        context 'when the rated disability has decision code NOTSVCCON in EVSS' do
+          let(:rated_disabilities_cassette) do
+            'evss/disability_compensation_form/rated_disabilities_with_non_service_connected'
+          end
```

Comment:
This is the one new spec in this file, corresponding to the new behavior. Note how I'm just using a `let` to swap out the usual `rated_disabilities` cassette for a nearly-identical one with a NOTSVCCON disability.